backend/common/settings-access.js

```diff
@@ -184,6 +184,16 @@ settings.getActiveChatUserListTimeout = function() {
     return inactiveTimer != null ? parseInt(inactiveTimer) : 10;
 };
 
+settings.getActiveMixplayUserList = function() {
+    let status = getDataFromFile("/settings/activeMixplayUsers/status");
+    return status != null ? status : "On";
+};
```

Comment:
I would change this to store a boolean instead of storing a string that serves the same purpose as a boolean (on/off is the same as true/false).
 I would also update the name to `settings.getActiveMixplayUserListEnabled` so its more clear what this means elsewhere (and is more in line with other newer settings that return booleans).

```js
settings.getActiveMixplayUserListEnabled = function() {
    let enabled = getDataFromFile("/settings/activeMixplayUsers/status");
    return status !== undefined ? enabled : true;
};
```